ansible/playbooks/AdoptOpenJDK_Linux_Playbook/group_vars/all/adoptopenjdk_variables.yml

```diff
@@ -2,12 +2,21 @@
 ##########################
 # adoptopenjdk_variables #
 ##########################
+
+# Jenkins User Variables:
+Jenkins_Username: jenkins
 Jenkins_User_SSHKey: /Vendor_Files/keys/id_rsa.pub
+
+# Superuser Variables:
+Superuser_Account: Enabled
 Zeus_User_SSHKey: /Vendor_Files/keys/zeus.key
-Nagios_User_SSHKey: /Vendor_Files/keys/nagios.key
-Jenkins_Username: jenkins
+
+# Nagios Variables:
 Nagios_Plugins: Enabled
-Slack_Notification: Disabled
-Superuser_Account: Enabled
+Nagios_User_SSHKey: /Vendor_Files/keys/nagios.key
+Nagios_Monitoring: Enabled
+Nagios_Master_IP: 78.47.239.96
```

Comment:
Actually this should be able to be set as `nagios.adoptopenjdk.net`. Makes it less work to update if the IP changes